backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointResource.java

```diff
@@ -515,16 +517,26 @@ public boolean unsubscribeEmail(
             )
     })
     @RolesAllowed(ConsoleIdentityProvider.RBAC_WRITE_INTEGRATIONS_ENDPOINTS)
-    public void testEndpoint(@Context SecurityContext sec, @RestPath UUID uuid) {
+    public void testEndpoint(@Context SecurityContext sec, @RestPath UUID uuid, @RequestBody EndpointTestRequest requestBody) {
```

Comment:
`requestBody` could be a `@NotBlank String` param. Is there any reason why we need the `EndpointTestRequest` class?

Reply:
I did it so that the following structure gets deserialized automatically:

```json
{
    "message": "Hello, World!"
}
```

Since otherwise I had to be using a JSON Object and what not...